components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/transactions.adoc

```diff
@@ -306,37 +294,71 @@ A `deploy_account` transaction has the following fields:
 
 |===
 
-=== Deploy account hash calculation
+=== v0 hash calculation
 
 The hash of a `deploy_account` transaction is computed as follows:
 
-[source,cairo]
+[source,cairo,subs="quotes"]
 ----
-deploy_account_tx_hash = ℎ(
+deploy_account_tx_hash = _h_(
     "deploy_account",
     version,
     contract_address,
     0,
-    ℎ(class_hash, contract_address_salt, constructor_calldata),
+    _h_(class_hash, contract_address_salt, constructor_calldata),
     max_fee,
     chain_id,
     nonce
 )
 ----
 
+#Missing parameters. Other txs have parameters here.#
+
+[id="deploy_transaction"]
+
+== `deploy` transaction hash calculation
+
+#Why do we inlcude the `deploy` transaction at all?#
```

Comment:
According @JameStark : If someone used the `deploy` transaction, they might need to retrieve the hash of it, and so this info is still important.